### ISSUE TITLE
Hyperparameter training command for protonet with WRN backbone on miniImagenet dataset

### ISSUE THREAD

**Issue by techNN** [Sha-Lab/FEAT#54]
Hi,
I appreciate your very insightful work as well as providing source code. Slightly related to Issue #24, I have been trying to recreate WRN pre-trained weights for miniImagenet that you had provided [here](https://drive.google.com/file/d/1fGE4s9aCUFBT75hNgEOSMbqfA3kANHiP/view) for the protonet but can't seem to get the 61.4% performance you obtained in those results. Would you be willing to provide the command you used to train that protonet WRN model? 

Thank you

**Comment by Han-Jia** [comment]
Hello,

For WRN, I use --batch_size 128 --lr 0.1 --dp_rate 0.3 --gamma 0.1 in pre-training. While in meta-training it uses the same command as with the ResNet backbone.

Given the pre-trained weights of WRN, please use the **--fix_bn** option to set the backbone to evaluation mode during training (which closes the dropout and stops updating the BN statistics) during the meta-training phase. We find --fix_bn is important for WRN.

**Comment by techNN** [comment]
Thank you, that is very helpful.
